example.ini

```diff
@@ -30,6 +34,11 @@ factory = reddit_service_activitygateway:make_wsgi_app
 metrics.namespace = activity_gateway
 metrics.endpoint =
 
+; request tracing
+tracing.enabled = true
```

Comment:
like on the puppet side, i think it'd be nicer to the user to just branch on whether or not the endpoint is defined or not rather than having a separate flag. this is consistent with the metrics client as well.